lib/widgets/product_listing.dart

```diff
@@ -53,18 +53,27 @@ class _ProductListingState extends State<ProductListing> {
   ) {
     List<Widget> widgets = List<Widget>();
 
-    widgets.add(_buildCategoryFilterMenu());
     snapshots.asMap().forEach((index, data) {
       _addProductToWidgetListInDisplay(data, widgets, context, index);
     });
 
-    return ListView(
+    return Column(
```

Comment:
Maybe add `crossAxisAlignment: CrossAxisAlignment.start` to ensure proper alignment in this column?